internal/criocli/criocli.go

```diff
@@ -128,8 +129,18 @@ func mergeConfig(config *libconfig.Config, ctx *cli.Context) error {
 			runtimeType := libconfig.DefaultRuntimeType
 			privilegedWithoutHostDevices := false
 			runtimeConfigPath := ""
+			var (
+				cmm int64
+				err error
+			)
 
 			switch len(fields) {
+			case 7:
+				cmm, err = units.RAMInBytes(fields[6])
+				if err != nil {
+					return fmt.Errorf("wrong value for container_min_memory: %q", fields[6])
```

Comment:
Just to keep this consistent with the other error.

```suggestion
					return fmt.Errorf("wrong value for --runtimes:container_min_memory: %q", fields[6])
```